js/lib/src/contract/matches.ts

```diff
@@ -538,6 +542,37 @@ export class MatchesInstruction {
       signers: [],
     };
   }
+
+  async resizeOracle(
+    args: ResizeOracleArgs,
+    _accounts = {},
+    _additionalArgs = {}
+  ) {
+    const [oracle, _oracleBump] = await getOracle(
+      new web3.PublicKey(args.seed),
+      args.authority
+    );
+
+    const match = (await getMatch(oracle))[0];
```

Comment:
You could rename this to `matchInstance`, would slightly simplify the accounts struct below.

Reply:
Don't forget about this one